agent/php_error.c

```diff
@@ -497,6 +593,23 @@ void nr_php_error_cb(int type,
     nr_txn_record_error(NRPRG(txn), nr_php_error_get_priority(type), msg,
                         errclass, stack_json);
 
+    /*
+     * Error Fingerprinting Callback
+     */
+    if (is_error_callback_set()) {
+#if ZEND_MODULE_API_NO < ZEND_8_1_X_API_NO
+      file = nr_strdup(error_filename);
+#else
+      file = nr_strdup(ZSTR_VAL(error_filename));
```

Comment:
I know the strdup of zvals is not handled consistently, but it's probably safer to use 
`file = nr_strdup(ZSTR_VAL(error_filename), ZSTR_LEN(error_filename));`

Reply:
Added in 1fe745c6faabb68875e513e71221b0eea8e49bc5